### PR TITLE
Add POA curl cron jobs

### PR DESCRIPTION
**Pull request by embiem**
This adds `curl` and POA related API calls via cron jobs.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 echo "$(date): backup service started"
 
-echo "${CRON_SCHEDULE}    /backup.sh" > /etc/crontabs/root
+echo "${CRON_SCHEDULE}    /scripts/backup.sh" > /etc/crontabs/root
+
+echo "*/7 * * * *    /scripts/cron_poa.sh" > /etc/crontabs/root
```
Comment (nicroto) [inline entrypoint.sh:7]: Maybe we can export those as env vars as well? (talking about the cron schedule expressions)

File: scripts/cron_watchtime.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+echo "$(date): sending POA watchtime request"
+
+curl -f -s -S --request POST \
+    --url "$POA_BASE_URL/api/twitch/updateWatchtime" \
+    --header "Authorization: Bearer $POA_TOKEN"
+
+curl -f -s -S --request POST \
+    --url $POA_HC_WATCHTIME_URL
```
Comment (nicroto) [inline scripts/cron_watchtime.sh:10]: Does this require a root url as well?

Comment (embiem) [reply scripts/cron_watchtime.sh:10]: No in this case I just set the whole URL as env var.

File: entrypoint.sh (hunk 1 above)
Comment (embiem) [reply entrypoint.sh]: true, just added.

**Pull request merged**